CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ This gem is moving onto its own [Semantic Versioning](https://semver.org/) schem
 
 Prior to v1.0.0 this gem was versioned based on the `MAJOR`.`MINOR` version of RuboCop. The first release of the ezcater_rubocop gem was `v0.49.0`.
 
+## v3.0.0
+- Upgrade rubocop:       1.16.0
+- Upgrade rubocop-rails: 2.10.1
+- Upgrade rubocop-rspec: 2.3.0
+- This is a major upgrade because a large number of cops have been introduced, tweaked, or renamed.
```

Comment:
Thoughts on including (or linking to) a list of renamed cops so that people can easily update their `rubocop:disable` statements?

Reply:
Sure.  Is the changelog the appropriate home for that?

Reply:
It's the thing that dependabot will excerpt in a bump PR and the first place someone should look when determining changes. If the list is short, it's probably fine to include it. If it's longer, it may make sense to create a separate file in a `docs/migrations/` folder and link to that, or link to an external resource that lives in the rubocop repos.

Reply:
I've rolled up the breaking changes from the three dependencies into our changelog.